src/setup/cortx_deploy

```diff
@@ -159,43 +131,12 @@ done
 
 export PATH=$PATH:/opt/seagate/provisioner/bin
 
-node_id=`cat /etc/machine-id`
-log $INFO "Starting provisioning on node $node_id."
-
-# Check current image version
-installed_version=$(conf_get $CORTX_CONFSTORE "node>$node_id>provisioning>version")
-release_version=$(conf_get $RELEASE_INFO "VERSION")
+# Apply Configuration
+apply_config
+[ ! -z $VALIDATE_CONF ] && validate_config
+# Deploy cortx
+deploy
```

Comment:
What about the upgrade CLI code. Please mark as TODO to be removed.

Reply:
done, Marked as TODO